<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>operator_Dialog</class>
 <widget class="QDialog" name="operator_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>月：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="mm_spinBox">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>日：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="yyyy_spinBox">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>21</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="dd_spinBox">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>年：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="query_pushButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>601</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="done_pushButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>252</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>任务完成</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_remindMission">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>621</width>
     <height>8</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835 - 副本.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>8</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>291</y>
     <width>621</width>
     <height>8</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835 - 副本.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>742</x>
     <y>0</y>
     <width>8</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1149835.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/backGround (4).jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>label_2</zorder>
  <zorder>mm_spinBox</zorder>
  <zorder>label_3</zorder>
  <zorder>yyyy_spinBox</zorder>
  <zorder>dd_spinBox</zorder>
  <zorder>label</zorder>
  <zorder>query_pushButton</zorder>
  <zorder>tableWidget</zorder>
  <zorder>done_pushButton</zorder>
  <zorder>label_remindMission</zorder>
  <zorder>label_7</zorder>
  <zorder>label_6</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
